net-libs/pjproject/pjproject-2.9.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools flag-o-matic
+
+DESCRIPTION="Open source SIP, Media, and NAT Traversal Library"
+HOMEPAGE="https://www.pjsip.org/"
+SRC_URI="https://www.pjsip.org/release/${PV}/${P}.tar.bz2"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Also I noted you dropped ~ppc, why is that?

Reply:
From the commit log:

https://archives.gentoo.org/gentoo-dev/message/ce73d8d5170b1f6ec04e19ea3bdae267

Final reply was this:

https://archives.gentoo.org/gentoo-dev/message/8b139b0c345da847f1f0bd95abb850fb

To which I received no objections.

Reply:
@juippis 

From my commit message:

    Drop ~ppc due to net-libs/bcg729 dep which is not ~ppc.

pkgcheck moaned about this.

Or can I somehow somehow force that if we're on PPC that USE=-g729 ?

I did note that bcg729 has keyword for ppc64 ... which begs the question:

1.  Can I just add ppc for bcg729?
2.  Can I switch to using ppc64 for pjproject?
3.  Can I enfore ARCH=ppc => USE=-g729?

Reply:
> Or can I somehow somehow force that if we're on PPC that USE=-g729 ?

Yes, it is the preferable way, because keywording and re-keywording packages are a lot of work.

>     1. Can I just add ppc for bcg729?

No, needs to be done by the arch team.

>     2. Can I switch to using ppc64 for pjproject?

Same as.

>     3. Can I enfore ARCH=ppc => USE=-g729?

Yes! It's under profiles/, have a look ;)

Reply:
package.use.mask updated in the pkgcheck referenced paths until it no longer moaned.  Please confirm that this is what you wanted me to get at and that this is now handled correctly please.